examples/Generator/generator_trainer.py

```diff
@@ -0,0 +1,176 @@
+import os
+from pathlib import Path
+
```

Comment:
Maybe some description what this file should showcase? When it is example it would be nice it it guides the user through what is going on. The name othe the file is not self-explanatory.

What is the differentce in purpose of inverse_trainer.py and generator_trainer.py ?

Reply:
Same as above.